packages/jaeger-ui/src/utils/redux-form-field-adapter.css

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright (c) 2017 Uber Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+input.ant-input.is-invalid {
+  border: rgba(255, 0, 0, 0.5) 1px solid;
+  background-color: rgba(255, 0, 0, 0.1);
+}
```

Comment:
Would suggest creating a `.SearchForm--validatedInput.is-invalid` style for this instead of overriding `.ant-input`, directly.

Also, this has a very strong aesthetic. What do you think of the following?

```
background: #fff9f8;
border: 1px solid #c00;
```

Reply:
Updated, this is far less aggressive.